react-native-fbsdkcore/js/FBSDKGraphRequest.ios.js

```diff
@@ -17,69 +17,71 @@
 * IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN
 * CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
 *
-* @providesModule FBSDKGraphRequest
 * @flow
 */
 
 'use strict';
 
-/*
+var FBSDKGraphRequestManager = require('./FBSDKGraphRequestManager.ios.js');
+
+type FBSDKGraphRequestCallback = (error: ?Object, result: ?Object) => void;
+type FBSDKGraphRequestParameters = {[key: string]: Object};
```

Comment:
Not sure if Flow can type that, but this parameters on native side seem to be expected in the following form:

```
{[key: string]: {string: ?Object, uri: ?Object}}
```